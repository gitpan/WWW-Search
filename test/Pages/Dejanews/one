http://www.deja.com/dnquery.xp?showsort=score&format=delta&defaultOp=OR&QRY=stuntboy&maxhits=100&ST=PS&
