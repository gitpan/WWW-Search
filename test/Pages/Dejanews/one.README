This set of test-result pages was created on Tue Jul 13 14:36:23 1999
