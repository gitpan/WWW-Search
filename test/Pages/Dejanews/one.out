http://x23.deja.com/getdoc.xp?AN=528086018&CONTEXT=939394692.1474953230&hitnum=0&fmt=raw
http://x23.deja.com/getdoc.xp?AN=514025828&CONTEXT=939394692.1474953230&hitnum=1&fmt=raw
http://x23.deja.com/getdoc.xp?AN=513619850&CONTEXT=939394692.1474953230&hitnum=2&fmt=raw
http://x23.deja.com/getdoc.xp?AN=411971686&CONTEXT=939394692.1474953230&hitnum=3&fmt=raw
http://x23.deja.com/getdoc.xp?AN=380878570&CONTEXT=939394692.1474953230&hitnum=4&fmt=raw
http://x23.deja.com/getdoc.xp?AN=380558291&CONTEXT=939394692.1474953230&hitnum=5&fmt=raw
http://x23.deja.com/getdoc.xp?AN=344239747&CONTEXT=939394692.1474953230&hitnum=6&fmt=raw
http://x23.deja.com/getdoc.xp?AN=457611381&CONTEXT=939394692.1474953230&hitnum=7&fmt=raw
http://x23.deja.com/getdoc.xp?AN=447336723&CONTEXT=939394692.1474953230&hitnum=8&fmt=raw
http://x23.deja.com/getdoc.xp?AN=430855626&CONTEXT=939394692.1474953230&hitnum=9&fmt=raw
http://x23.deja.com/getdoc.xp?AN=412274025&CONTEXT=939394692.1474953230&hitnum=10&fmt=raw
http://x23.deja.com/getdoc.xp?AN=404097303&CONTEXT=939394692.1474953230&hitnum=11&fmt=raw
http://x23.deja.com/getdoc.xp?AN=391526876&CONTEXT=939394692.1474953230&hitnum=12&fmt=raw
http://x23.deja.com/getdoc.xp?AN=380890731&CONTEXT=939394692.1474953230&hitnum=13&fmt=raw
http://x23.deja.com/getdoc.xp?AN=357170716&CONTEXT=939394692.1474953230&hitnum=14&fmt=raw
http://x23.deja.com/getdoc.xp?AN=344633700&CONTEXT=939394692.1474953230&hitnum=15&fmt=raw
http://x23.deja.com/getdoc.xp?AN=411826948&CONTEXT=939394692.1474953230&hitnum=16&fmt=raw
http://x23.deja.com/getdoc.xp?AN=344519790&CONTEXT=939394692.1474953230&hitnum=17&fmt=raw
http://x23.deja.com/getdoc.xp?AN=319162328&CONTEXT=939394692.1474953230&hitnum=18&fmt=raw
http://x23.deja.com/getdoc.xp?AN=473928671&CONTEXT=939394692.1474953230&hitnum=19&fmt=raw
http://x23.deja.com/getdoc.xp?AN=473778429&CONTEXT=939394692.1474953230&hitnum=20&fmt=raw
http://x23.deja.com/getdoc.xp?AN=358707225&CONTEXT=939394692.1474953230&hitnum=21&fmt=raw
http://x23.deja.com/getdoc.xp?AN=358728767&CONTEXT=939394692.1474953230&hitnum=22&fmt=raw
http://x23.deja.com/getdoc.xp?AN=358773325&CONTEXT=939394692.1474953230&hitnum=23&fmt=raw
http://x23.deja.com/getdoc.xp?AN=290386084&CONTEXT=939394692.1474953230&hitnum=24&fmt=raw
http://x23.deja.com/getdoc.xp?AN=224274981&CONTEXT=939394692.1474953230&hitnum=25&fmt=raw
http://x23.deja.com/getdoc.xp?AN=224110582&CONTEXT=939394692.1474953230&hitnum=26&fmt=raw
http://x23.deja.com/getdoc.xp?AN=224036967&CONTEXT=939394692.1474953230&hitnum=27&fmt=raw
http://x23.deja.com/getdoc.xp?AN=224041824&CONTEXT=939394692.1474953230&hitnum=28&fmt=raw
http://x23.deja.com/getdoc.xp?AN=310495917&CONTEXT=939394692.1474953230&hitnum=29&fmt=raw
http://x23.deja.com/getdoc.xp?AN=271323003&CONTEXT=939394692.1474953230&hitnum=30&fmt=raw
http://x23.deja.com/getdoc.xp?AN=238580563&CONTEXT=939394692.1474953230&hitnum=31&fmt=raw
http://x23.deja.com/getdoc.xp?AN=238684898&CONTEXT=939394692.1474953230&hitnum=32&fmt=raw
http://x23.deja.com/getdoc.xp?AN=238695078&CONTEXT=939394692.1474953230&hitnum=33&fmt=raw
http://x23.deja.com/getdoc.xp?AN=238410107&CONTEXT=939394692.1474953230&hitnum=34&fmt=raw
http://x23.deja.com/getdoc.xp?AN=224032657&CONTEXT=939394692.1474953230&hitnum=35&fmt=raw
http://x23.deja.com/getdoc.xp?AN=224034321&CONTEXT=939394692.1474953230&hitnum=36&fmt=raw
http://x23.deja.com/getdoc.xp?AN=271779485&CONTEXT=939394692.1474953230&hitnum=37&fmt=raw
http://x23.deja.com/getdoc.xp?AN=271354154&CONTEXT=939394692.1474953230&hitnum=38&fmt=raw
http://x23.deja.com/getdoc.xp?AN=238573483&CONTEXT=939394692.1474953230&hitnum=39&fmt=raw
http://x23.deja.com/getdoc.xp?AN=238591681&CONTEXT=939394692.1474953230&hitnum=40&fmt=raw
http://x23.deja.com/getdoc.xp?AN=238656889&CONTEXT=939394692.1474953230&hitnum=41&fmt=raw
http://x23.deja.com/getdoc.xp?AN=238662468&CONTEXT=939394692.1474953230&hitnum=42&fmt=raw
http://x23.deja.com/getdoc.xp?AN=238724888&CONTEXT=939394692.1474953230&hitnum=43&fmt=raw
http://x23.deja.com/getdoc.xp?AN=238729815&CONTEXT=939394692.1474953230&hitnum=44&fmt=raw
http://x23.deja.com/getdoc.xp?AN=224118475&CONTEXT=939394692.1474953230&hitnum=45&fmt=raw
http://x23.deja.com/getdoc.xp?AN=107893748&CONTEXT=939394692.1474953230&hitnum=46&fmt=raw
http://x23.deja.com/getdoc.xp?AN=247244892&CONTEXT=939394692.1474953230&hitnum=47&fmt=raw
http://x23.deja.com/getdoc.xp?AN=239824214.2&CONTEXT=939394692.1474953230&hitnum=48&fmt=raw
http://x23.deja.com/getdoc.xp?AN=138349042&CONTEXT=939394692.1474953230&hitnum=49&fmt=raw
http://x23.deja.com/getdoc.xp?AN=138845192&CONTEXT=939394692.1474953230&hitnum=50&fmt=raw
http://x23.deja.com/getdoc.xp?AN=145048106&CONTEXT=939394692.1474953230&hitnum=51&fmt=raw
http://x23.deja.com/getdoc.xp?AN=144142775&CONTEXT=939394692.1474953230&hitnum=52&fmt=raw
http://x23.deja.com/getdoc.xp?AN=138639977&CONTEXT=939394692.1474953230&hitnum=53&fmt=raw
http://x23.deja.com/getdoc.xp?AN=138286099&CONTEXT=939394692.1474953230&hitnum=54&fmt=raw
